kernel/include/kswap.h

```diff
@@ -24,8 +24,11 @@ extern struct k_spinlock sched_spinlock;
  * thread (via either z_swap() or interrupt/exception return!) we need
  * to restore the lock state to whatever the thread's counter
  * expects.
+ *
+ * Note: The paired function z_smp_reacquire_global_lock() has been
+ * removed due to no longer being used during context switch.
+ * See issue #33273.
```

Comment:
Take this comment out.   There's no need to document functions that don't exist.

Reply:
Thank you for correcting this,  @andyross, got it fix~ ^^